apps/ui/src/components/SpaceCover.vue

```diff
@@ -6,22 +6,24 @@ import { NetworkID } from '@/types';
 
 const props = defineProps<{
   space: { id: string; cover: string; avatar: string; network: NetworkID };
+  size: 'sm' | 'lg';
```

Comment:
It should be marked as optional with default fallback, right now when we use this component somewhere without passing it it doesn't pass type check so CI is failing as well (or we should always pass it as prop if it makes more sense).